Header header

float64 thetaMinFreeSector
Centroid thetaMin_Centroid
float64 thetaMaxFreeSector
Centroid thetaMax_Centroid
